core/src/components/alert/common.ts

```diff
@@ -174,16 +184,40 @@ export function createCommonAlert(config?: PropsConfig<CommonAlertProps>): Commo
 
 	const visible$ = transition.stores.visible$;
 	const hidden$ = transition.stores.hidden$;
-	return {
+
+	const widget: CommonAlertWidget = {
 		...stateStores({...stateProps, visible$, hidden$}),
 		patch,
 		api: {
 			open,
 			close,
 		},
 		directives: {
+			containerDirective: mergeDirectives(
+				createAttributesDirective(() => ({
+					attributes: {
+						role: 'alert',
+						class: stateProps.className$,
+					},
+					classNames: {
+						'au-alert': true,
+					},
+				})),
+				bindDirectiveNoArg(transition.directives.directive),
+			),
 			transitionDirective: bindDirectiveNoArg(transition.directives.directive),
+			dismissibleDirective: createAttributesDirective(() => ({
+				events: {
+					click: widget.api.close,
+				},
+				attributes: {
+					'aria-label': stateProps.ariaCloseButtonLabel$,
+					type: 'button',
```

Comment:
Yeah, I mean, many directives will be questionable then, apart from the ones adding aria attributes.

If you see I also left `transitionDirective`, I would expected a comment there because it is no longer used by our widget by default, because containerDirective does a `mergeDirectives`.

On one side we simplify the implementation if you so an implementation like we thought for our bootstrap, on the other we impose stuff with directives that an headless user may not be interested in.